letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -380,13 +424,14 @@ def get_virtual_hosts(self):
         return vhs
 
     def is_name_vhost(self, target_addr):
-        r"""Returns if vhost is a name based vhost
+        """Returns if vhost is a name based vhost
 
         NameVirtualHost was deprecated in Apache 2.4 as all VirtualHosts are
         now NameVirtualHosts. If version is earlier than 2.4, check if addr
         has a NameVirtualHost directive in the Apache config
 
-        :param str target_addr: vhost address ie. \*:443
+        :param target_addr: vhost address
+        :type target_addr: :class:~letsencrypt_apache.obj.Addr
```

Comment:
Forgot the backticks... 

FTR - all code with user defined classes used to use the ```:type arg: :class:....``` I was just trying to be consistent.  
I will begin to move to the shorthand version.